src/main/java/org/sagebionetworks/web/client/widget/discussion/DiscussionThreadWidget.java

```diff
@@ -99,14 +101,13 @@ public void toggleThread() {
 
 	@Override
 	public void toggleReplies() {
-		if (!areRepliesConfigure) {
-			configureReplies();
-		}
+		configureReplies();
 		view.toggleReplies();
 	}
 
 	public void configureReplies() {
 		synAlert.clear();
+		view.clearReplies();
 		offset = 0L;
 		if (order == null) {
 			order = DEFAULT_ORDER;
```

Comment:
the getRepliesForThread is going to need to be in another method once you hook up the onLoadMoreReplies() action.